website/src/components/SearchPage/DownloadDialog/ActiveDownloadFilters.tsx

```diff
@@ -1,33 +1,58 @@
 import type { FC } from 'react';
 
-import type { FieldValues } from '../../../types/config';
+import type { DownloadParameters } from './DownloadParameters';
 
 type ActiveDownloadFiltersProps = {
-    lapisSearchParameters: Record<string, any>;
-    hiddenFieldValues: FieldValues;
+    downloadParameters: DownloadParameters;
 };
 
-export const ActiveDownloadFilters: FC<ActiveDownloadFiltersProps> = ({ lapisSearchParameters, hiddenFieldValues }) => {
-    let filterValues = Object.entries(lapisSearchParameters)
-        .filter((vals) => vals[1] !== undefined && vals[1] !== '')
-        .filter(([name, val]) => !(Object.keys(hiddenFieldValues).includes(name) && hiddenFieldValues[name] === val))
-        .map(([name, filterValue]) => ({ name, filterValue: filterValue !== null ? filterValue : '' }));
+export const ActiveDownloadFilters: FC<ActiveDownloadFiltersProps> = ({ downloadParameters }) => {
+    let badges = null;
+    const badgeClasses = 'border-black border rounded-full px-2 py-1 text-sm';
 
-    filterValues = filterValues.filter(({ filterValue }) => filterValue.length > 0);
-    if (filterValues.length === 0) {
-        return null;
+    switch (downloadParameters.type) {
+        case 'filter': {
+            let filterValues = Object.entries(downloadParameters.lapisSearchParameters)
+                .filter((vals) => vals[1] !== undefined && vals[1] !== '')
+                .filter(
+                    ([name, val]) =>
+                        !(
+                            Object.keys(downloadParameters.hiddenFieldValues).includes(name) &&
+                            downloadParameters.hiddenFieldValues[name] === val
+                        ),
+                )
+                .map(([name, filterValue]) => ({ name, filterValue: filterValue !== null ? filterValue : '' }));
+
+            filterValues = filterValues.filter(({ filterValue }) => filterValue.length > 0);
+
+            if (filterValues.length > 0) {
+                badges = filterValues.map(({ name, filterValue }) => (
+                    <div key={name} className={badgeClasses}>
+                        {name}: {typeof filterValue === 'object' ? filterValue.join(', ') : filterValue}
+                    </div>
+                ));
+            }
+            break;
+        }
+        case 'select': {
+            const count = downloadParameters.selectedSequences.size;
+            if (count > 0) {
+                badges = (
+                    <div className={badgeClasses}>
+                        {count.toLocaleString()} sequence{count === 1 ? '' : 's'} selected
```

Comment:
I think the plan was to put this in the Download button? https://loculus.slack.com/archives/C05G172HL6L/p1726830336921539?thread_ts=1726828351.088969&cid=C05G172HL6L

In particular - the selection UI IMO adds confusion about how to download all sequences so adjusting the text of this button would be useful for addressing that too

Reply:
Ah yes, sorry for not taking that into account.

But which "Download" button should have the changed text? The one in the modal, right? Or the one that opens the modal?

Reply:
The one that opens the modal is the main important one IMO

Reply:
I'll add it. But I think I'd keep the "Active filter" bit in there, also the 'showing xx entries, y selected' or should I remove that again? I think it doesn't hurt. Especially in the open modal it's  nice to see that the selection overrides the other filters, and to see how many have been selected.

Reply:
Keeping the filter sounds great. Yeah I think I agree it doesn't hurt (it will be easier to see in a demo whether it feels odd to see the info twice). I find the comma a bit throwing especially because the numbers can be formatted with commas. I think I would ideally see it as `23,123 sequences (3 selected)`. Potentially with an X in the `3 selected to clear`. But we can do that in a future PR if you prefer.

Reply:
Alright, i've kept the filter but indeed after trying it out it was odd to see the same info twice.

Button text is now in line with what we discussed in Slack!

https://github.com/loculus-project/loculus/pull/2917/commits/b1675cd578809dfb6fd1f8b3e2809e215c7d2c19